scripts/scil_fodf_bundleparc.py

```diff
@@ -3,17 +3,17 @@
 """
 BundleParc: automatic tract labelling without tractography.
 
-This method takes as input fODF maps of order 6 (or above) and a whole-brain WM mask and outputs 71 bundle label maps. These maps can then be used to perform tractometry/tract profiling/radiomics. The bundle definitions follow TractSeg's minus the whole CC.
+This method takes as input fODF maps and outputs 71 bundle label maps. These maps can then be used to perform tractometry/tract profiling/radiomics. The bundle definitions follow TractSeg's minus the whole CC.
 
 Example usage:
-    $ scil_fodf_bundleparc.py fodf.nii.gz wm_mask.nii.gz --out_prefix sub-001__
+    $ scil_fodf_bundleparc.py fodf.nii.gz --out_prefix sub-001__
 
 Example output:
     sub-001__AF_left.nii.gz, sub-001__AF_right.nii.gz, ..., sub-001__UF_right.nii.gz
 
 The output can be further processed with scil_bundle_mean_std.py to compute statistics for each bundle.
 
-This script requires PyTorch to be installed. To install it, see the official website: https://pytorch.org/get-started/locally/
+This script requires PyTorch==2.2.1 to be installed. See the official website: https://pytorch.org/get-started/locally/
 
 This script requires a GPU with ~6GB of available memory. If you use
```

Comment:
Does this work with CPU too, my laptop seems to be crashing when using this (but not my desktop).

Reply:
Yes, you need to install the right version though. How does it crash ?